frontend/src/Components/Link/IconButton.js

```diff
@@ -23,6 +23,7 @@ function IconButton(props) {
         className,
         isDisabled && styles.isDisabled
       )}
+      name={name}
```

Comment:
`name` is an object, I don't think we want to pass that as the name attribute on the button

Reply:
Fixed